docs/Developer Documentation/ResumeController/index.md

```diff
@@ -0,0 +1,224 @@
+# Resume Controller
+
+This page will descrive internal structure and detailed design of Resume controller 
+
+|||
+Figure 1: ResumeController Overview
```

Comment:
@LuxoftAKutsan please check the markdown, it looks like 

> ||| Figure 1: ResumeController Overview

Reply:
Literally it is not a markdown. This will be generated to HTML page. Same as guidelines. Take an example of https://github.com/LuxoftSDL/sdl_core_guides/blob/master/docs/Developer Documentation/Transport Manager/index.md and https://smartdevicelink.com/en/guides/core/developer-documentation/transport-manager/